src/MedicalCard/dateFormat.js

```diff
@@ -0,0 +1,21 @@
+const monthNames = [
```

Comment:
should it be prepared for localization?

Reply:
Cool so translations are next on my todo list after some minor style changes in the emergency service. I’ll start here for translations with another PR to make these general.